R/spliceHelperFunctions.R

```diff
@@ -96,232 +96,180 @@ findSpliceOverlapsByDist <-function(query, subject, ignore.strand=FALSE, maxDist
   olap
 }
 
+#' annotate splice overlap by distance
+#' @noRd
 annotateSpliceOverlapsByDist <-function(query, subject) {
-
-  ## examples for test purposes
```

Comment:
this could be implemented as a unit test

Reply:
A unit test is implemented with query and subject stored as rds objects.